src/hooks/useHandleOnScrollCallback/index.ts

```diff
@@ -1,25 +1,25 @@
 import React, { useCallback } from 'react';
-import { SCROLL_BUFFER } from '../../../../utils/consts';
+import { SCROLL_BUFFER } from '../../utils/consts';
 
 export interface UseHandleOnScrollCallbackProps {
-  setShowScrollDownButton: React.Dispatch<React.SetStateAction<boolean>>;
   hasMore: boolean;
   onScroll(fn: () => void): void;
   scrollRef: React.RefObject<HTMLDivElement>;
+  setShowScrollDownButton?: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
 export function calcScrollBottom(scrollHeight: number, scrollTop: number): number {
   return scrollHeight - scrollTop
 }
 
 export function useHandleOnScrollCallback({
-  setShowScrollDownButton,
   hasMore,
   onScroll,
   scrollRef,
-}: UseHandleOnScrollCallbackProps): (e: React.UIEvent<HTMLElement>) => void {
-  return useCallback((e) => {
-    const element = e.target as Element;
```

Comment:
I thought we don't need to get an event param here since we can use the scrollRef instead. I checked the behavior in OpenChannel \w Hoon to make sure I didn't break anything made by https://github.com/sendbird/sendbird-uikit-react/pull/460. 

Plus, to replace the scrollIntoView in useEffect or useLayoutEffect \w this custom hook, there was no way to get the event param. That's why I had to find other way to make things work.